plugins/services/src/js/structs/Framework.js

```diff
@@ -1,15 +1,28 @@
 import { cleanServiceJSON } from "#SRC/js/utils/CleanJSONUtil";
 import { isSDKService } from "#PLUGINS/services/src/js/utils/ServiceUtil";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
 
 import {
   ROUTE_ACCESS_PREFIX,
   FRAMEWORK_ID_VALID_CHARACTERS
 } from "../constants/FrameworkConstants";
 import FrameworkUtil from "../utils/FrameworkUtil";
+import * as ServiceStatus from "../constants/ServiceStatus";
 
 import Application from "./Application";
 import FrameworkSpec from "./FrameworkSpec";
 
+const getHighestPriorityStatus = tasks => {
+  const statuses = tasks
+    .map(t => findNestedPropertyInObject(t, "checkResult.http.statusCode"))
+    .map(ServiceStatus.fromHttpCode)
+    .filter(x => x);
```

Comment:
Maybe we should rename this to something with more meaning.

Reply:
like `.filter(isTruthy => isTruthy)` or `.filter(status => status)`?
could you pick your favourite or make a suggestion please, as i hardly see added value here.

Reply:
Sorry I mend more describing to what is in the variable my suggestion would be status.

Reply:
done